compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -1502,11 +1891,46 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
 
                 let l = self.new_named_lifetime(lifetime.id, id, span, ident);
                 hir::GenericArg::Lifetime(l)
-            }));
+            })
+            .collect();
         debug!(?lifetimes);
 
+        let type_const_args = self.collect_type_const_args(&ast_generics.params);
+
+        let universal_args: Vec<_> = impl_trait_inputs.iter().map(|ty| match ty.kind {
+            TyKind::ImplTrait(node_id, _) => {
+                let def_id = self.local_def_id(node_id);
+                let span = ty.span;
```

Comment:
```suggestion
                let span = self.lower_span(ty.span);
```
so you don't need to call it again, not `lower_ident` with the same span.